fudge-ratio/src/main/webapp/code/mainScreen.js

```diff
@@ -37,8 +37,9 @@ const showMain = () =>
     const text = get("timer-display");
     const startStop = get("start-stop");
     const finish = get("finish");
-
+    const oneDay = 1000*60*60*24;
```

Comment:
This in another level of nitpicking but:
`` const oneDay = 1000 * 60 * 60 * 24;``